ui-frontend/packages/catalog-ui-search/src/main/webapp/js/model/Query.js

```diff
@@ -123,11 +123,7 @@ Query.Model = Backbone.AssociatedModel.extend({
       typeof data.filterTree === 'string'
     ) {
       // for backwards compatability
-      try {
-        data.filterTree = JSON.parse(data.filterTree)
-      } catch (e) {
-        data.filterTree = CQLUtils.transformCQLToFilter(data.cql)
-      }
+      data.filterTree = new FilterBuilderClass(JSON.parse(data.filterTree))
```

Comment:
❓ Should we be concerned with the backwards compatibility with cql that's mentioned here?

Reply:
@andrewkfiedler would know. I don't believe so. My understanding is that the filterTree being passed in, not as a FilterBuilderClass, is the backwards compatibility we're concerned with.